backend/core/types/src/backend-swagger.ts

```diff
@@ -4998,13 +5018,13 @@ export enum UserRole {
   "admin" = "admin",
 }
 export enum EnumListingFilterParamsComparison {
-  "=" = "=",
-  "<>" = "<>",
+  '=' = '=',
+  '<>' = '<>'
 }
-export enum EnumListingFilterParamsStatus {
-  "active" = "active",
-  "pending" = "pending",
-  "closed" = "closed",
+export enum ListingFilterKeys {
+  'status' = 'status',
+  'name' = 'name',
+  'neighborhood' = 'neighborhood',
 }
```

Comment:
Ditto for these. Something about my annotations on the listings controller GET endpoint are incorrect. I had to manually include these types because Swagger didn't.

Reply:
Updated based on @pbn4's comments below